src/app/templates/play-control.tpl

```diff
@@ -0,0 +1,36 @@
+<div class="play-control">
```

Comment:
ok, this is split in "component", great, but there is no css attached to it. Shouldnt there be a play-control.styl?